cmd/unsub/server.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"github.com/willf/bloom"
+)
+
+// *** GLOBAL VARIABLES ***
+
+var bloomFilter *bloom.BloomFilter

```

Comment:
There is no need for global variables. I would recommend creating a bloom filter type and using methods.
See https://tour.golang.org/methods/4
You would have something like:
```
type bloomFilter struct {
   filter *bloom.BloomFilter
}

func newBloomFilter() *bloomFilter {
   return &bloomFilter{
       filter: bloom.New(...),
   }
}

func (bf *bloomFilter) check() {
...
}

func (bf *bloomFilter) populate() {
...
}
```